main.js

```diff
@@ -155,70 +155,34 @@ document.getElementById("counter-rooms-minus").onclick = () => {
 
 const urlGet = 'https://fe-student-api.herokuapp.com/api/hotels'
 const btnSearchHeader = document.getElementById('btnSearchHeader')
-
 btnSearchHeader.addEventListener('click', (event) => {
     event.preventDefault()
-    fetch(urlGet)
+    let search = document.getElementById('searchHeader').value
+    let adults = document.getElementById('adults-value').value
+    let children = document.getElementById('children-value').value
+    let rooms = document.getElementById('rooms-value').value
+
+    let isValid = false;
```

Comment:
So, if you use "return" when validation failed you can remove this variable. The reason was that we needed to block fetch it validation failed